test/unit/services/test-get-caseload.js

```diff
@@ -24,7 +25,12 @@ const CASELOAD = {
   a: 0
 }
 
-const TEAM_CASELOAD = [CASELOAD, CASELOAD]
+const OVERALL_CASELOAD = Object.assign({}, CASELOAD, {totalCases: 9, untiered: 3, d2: 3, c1: 3, b1: 3})
+const COMMUNITY_CASELOAD = Object.assign({}, CASELOAD, {caseType: 'COMMUNITY'})
+const CUSTODY_CASELOAD = Object.assign({}, CASELOAD, {caseType: 'CUSTODY'})
+const LICENSE_CASELOAD = Object.assign({}, CASELOAD, {caseType: 'LICENSE'})
```

Comment:
We already have the default values in the CASELOAD object. The overall alters the default values by summing those default values.